### PR TITLE
Dev milestone4

### PR DESCRIPTION
**Pull request by Wusinho**
For this part of the project, the logic and the calculations are included.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    return num1.div(num2).toString();
+  } if (operation === 'X') {
+    return num1.times(num2).toString();
+  } if (operation === '+') {
+    return num1.plus(num2).toString();
+  } if (operation === '-') {
+    return num1.minus(num2).toString();
+  } if (operation === '%') {
+    return num1.mod(num2).toString();
+  }
```
Comment (uche-inyama) [inline src/logic/operate.js]: In this operation the implementation is like so:
```
if (operation === '%'){
  return num1.div(100).toString()
}
```

File: src/components/App.js [hunk 2]
```diff
@@ -1,14 +1,35 @@
 import React from 'react';
-import ButtonPanel from './ButtonPanel';
 import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
+// import calculate from '../logic/calculate';
 import '../assets/styles/App.scss';
```
Comment (uche-inyama) [inline src/components/App.js]: Please remove the commented code.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required :recycle: 

### Hi Juan :slightly_smiling_face: 

There are changes to make and I have left an inline comment below. 

Make the changes, then submit another form for a code review.

### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Comment by rukundoeric** [comment]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - Follows gitwork-flow.  💯
  - Codebase looks clean and professional. 💯
  - Issues raised from the previous review are all addressed. 💯 
  

- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/),
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**